pages/product.html

```diff
@@ -0,0 +1,429 @@
+        class="product-service__blur-background"
+      />
+      <div class="product-service__card-container">
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/security-solid.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Security and privacy</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/neural-network-icon.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Built-in AI features</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/collaborate-icon.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Collaborate with others</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/memory-card-icon.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Real time sync</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/nlp.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Built-in AI features</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+        <div class="product-service__card">
+          <img
+            src="../assets/vectors/note-organization-icon.svg"
+            class="product-service__card-icon"
+          />
+          <h3>Easy notes organization</h3>
+          <p class="product-service__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy eirmod tempor.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="prodcut-marketing-insight">
+      <div class="prodcut-marketing-insight__card">
+        <div class="product-marketing-insight__card-body">
+          <h4 class="product-marketing-insight__card-title">
```

Comment:
`product-marketing-insight__card-body` `product-marketing-insight__card-title` you are using element outside its block